client/lib/helpers/responsiveness.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class Responsive extends StatelessWidget {
+  final Widget mobile;
+  final Widget? tablet;
+  final Widget desktop;
+
+  const Responsive({
+    Key? key,
+    required this.mobile,
+    this.tablet,
```

Comment:
Please ensure to always have required elements at the beginning of the constructor